systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -1985,8 +1897,93 @@ void testUOListeningOnlyUsersInSameCluster() {
         assertThat(kafkaUserResource, containsString("strimzi.io/cluster: " + firstClusterName));
     }
 
+    @Test
+    void testMessagesAreStoredInDisk() throws Exception {
+        testMethodResources().kafkaEphemeral(CLUSTER_NAME, 3, 1)
+            .editSpec()
+                .editKafka()
+                    .editListeners()
+                        .withNewKafkaListenerExternalNodePort()
+                            .withTls(false)
+                        .endKafkaListenerExternalNodePort()
+                    .endListeners()
+                .endKafka()
+            .endSpec()
+            .done();
+
+        testMethodResources().topic(CLUSTER_NAME, TEST_TOPIC_NAME, 3, 1).done();
```

Comment:
With 3 node Kafka cluster, should we use 3 replicas here to make sure the partition scheduling works? Or the other way around, should we use just single node cluster?

Reply:
I changed it to single node cluster.